webservices/cxf/src/java/org/sakaiproject/webservices/SakaiScript.java

```diff
@@ -4257,4 +4259,111 @@ private void setToolProperties(ToolConfiguration tool, String propList) {
         }
     }
 
+    @WebMethod
+    @Path("/getSessionCountForServer")
+    @Produces("text/plain")
+    @GET
+    public Integer getSessionCountForServer(
+            @WebParam(name = "sessionid", partName = "sessionid") @QueryParam("sessionid") String sessionid,
+            @WebParam(name = "serverid", partName = "serverid") @QueryParam("serverid") String serverid,
+            @WebParam(name = "millisBeforeExpire", partName = "millisBeforeExpire") @QueryParam("millisBeforeExpire") int millisBeforeExpire) {
+        //register the session with presence
+        Session s = establishSession(sessionid);
```

Comment:
Yes, I think we should add only superuser access here. Thanks for the comment!  I will review the other webservices we've sent, to check if any of these needs that too.